webapp/platform/client/src/client4.ts

```diff
@@ -1747,7 +1747,7 @@ export default class Client4 {
         );
     };
 
-    getChannelStats = (channelId: string, excludeFilesCount = false) => {
+    getChannelStats = (channelId: string, excludeFilesCount = true) => {
```

Comment:
2/5 that it makes it more clear to the consumer of this method that the file count is expensive to fetch if the flag is inverted. 
```suggestion
    getChannelStats = (channelId: string, includeFilesCount = false) => {
```

Reply:
Agree naming is confusing here, but if we change the name as you suggest we would need to modify several thing in the BE too. I know the name is not the best, but for keep this simple, we should continue with the original name used and with the boolean values making sense with the variable name.

Reply:
Do we really need to also touch the backend? This could be a client-only change

Reply:
the way you place it to say, includeFilesCount, it then should be false, then yes, we would need to touch the BE to accomodate it to the new values, because it was initially thought to be focus on always fetch the fileCount so the only option was to exclude.